options.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'app'
+
```

Comment:
- [ ] As you guys know, this is a pair-programming project but i cannot see any commit from the second member of the team 🤔 Kindly ask your partner to make a commit in this pull request 😊